### DISCUSSION TITLE
Outdated firmware.patch

### DISCUSSION THREAD

**Discussion by QBANIN** [Koenkk/zigbee2mqtt#11618] [General]
Provided firmware.patch from dev branch doesn't work for simplelink_cc13xx_cc26xx_sdk_5_40_00_40 and thereofor for example  preinclude.h is not being created

> git apply  firmware.patch --ignore-space-change
> 
> firmware.patch:284: trailing whitespace.
>   
> firmware.patch:286: trailing whitespace.
>   // ZDSecMgrPermitJoining is never called. Joining via a router would always fail now since 
> firmware.patch:1240: trailing whitespace.
>     // Set TXRX pin to 0 in RX and high impedance in idle/TX. 
> firmware.patch:1258: trailing whitespace.
>     // Set TXRX pin to 0 in RX/TX and high impedance in idle. 
> firmware.patch:1459: new blank line at EOF.
> +
> error: łatanie nie powiodło się: znp_CC1352P_2_LAUNCHXL_tirtos_ccs/znp.syscfg:4
> error: znp_CC1352P_2_LAUNCHXL_tirtos_ccs/znp.syscfg: łatka się nie stosuje
> error: łatanie nie powiodło się: znp_CC26X2R1_LAUNCHXL_tirtos_ccs/znp.syscfg:47
> error: znp_CC26X2R1_LAUNCHXL_tirtos_ccs/znp.syscfg: łatka się nie stosuje
> error: łatanie nie powiodło się: znp_LP_CC2652RB_tirtos_ccs/znp.syscfg:48
> error: znp_LP_CC2652RB_tirtos_ccs/znp.syscfg: łatka się nie stosuje
>

**Comment by Koenkk** [comment]
dupe of https://github.com/Koenkk/zigbee2mqtt/issues/11619
